charts/kuberpult/values.yaml

```diff
@@ -61,6 +61,9 @@ argocd:
 datadogTracing:
   enabled: false
   debugging: false
-  metrics: false
```

Comment:
Q: This was unused before?

Reply:
it's related to my new story, I added it but didn’t adapt the helm chart with it, I did that in this PR so I just reorganized it